config/formats.ts

```diff
@@ -1278,7 +1278,7 @@ export const Formats: FormatList = [
 
 			const mixedSpecies = this.dex.deepClone(species);
 			mixedSpecies.weightkg =
-				Math.max(0.1, +(species.weightkg + crossSpecies.weightkg - crossPrevoSpecies.weightkg)).toFixed(1);
+				parseInt(Math.max(0.1, +(species.weightkg + crossSpecies.weightkg - crossPrevoSpecies.weightkg)).toFixed(1));
```

Comment:
Does Math.max not return a `number`?

Reply:
The `toFixed(1)` changes it to a `string`, but it wasn't being picked up b/c `mixedSpecies` was `any`, and I guess JS was silently converting `string` to `number` in calculations.